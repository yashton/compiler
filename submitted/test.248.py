def wowhat(i,jh,sk 
sdf,wg)
