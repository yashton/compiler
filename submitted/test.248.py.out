(KEYWORD def)
(ID "wowhat")
(PUNCT "(")
(ID "i")
(ERROR "Illegal character ',jh,sk 
sdf,wg)
' on line 1)
