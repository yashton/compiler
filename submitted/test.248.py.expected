(KEYWORD def)
(ID "wowhat")
(PUNCT "(")
(ID "i")
(PUNCT ",")
(ID "jh")
(PUNCT ",")
(ID "sk")
(ID "sdf")
(PUNCT ",")
(ID "wg")
(PUNCT ")")
(NEWLINE)
(ENDMARKER)
